movies/src/main/java/com/fundacionjala/movies/MovieRegular.java

```diff
@@ -0,0 +1,33 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Charito on 6/10/2016.
+ */
+public class MovieRegular extends Movie {
+    public static final int REGULAR = 0;
+
+    private static final double PRICE_RENT = 2;
+
+    private static final int delay = 2;
```

Comment:
this variable should be in uppercase.
I think this variable should be called DAYS_ALLOWED
```java
private static final int DAYS_ALLOWED = 2;
```